plugins/csp-vr-accessibility/README.md

```diff
@@ -14,27 +14,31 @@ This plugin can be enabled with the following configuration in your `settings.js
-      "vignetteFadeDuration": <float>, // The time it takes for the animation to fade in the vignette in seconds.
-      "vignetteFadeDeadzone": <float>, // The time of movement above the velocity threshold that is needed before the animation is played in seconds.
-      "vignetteLowerVelocityThreshold": <float>, // The lower threshold below which the vignette will not display (relative to Spice frame from 0..~10). 
-      "vignetteUpperVelocityThreshold": <float>, // The upper threshold above which the vignette is always at ist minimum radius (relative to Spice frame from 0..~10).
-      "vignetteUseDynamicRadius": <bool>, // Toggle whether to use the dynamic vignette radius, or the fade animation with fixed radii.
-      "vignetteUseVerticalOnly": <bool> // Toggle whether to only draw the vignette horizontally and keep the sides unobstructed.
+      "grid": {
+        "enabled": bool,   // Toggle whether the Grid should be visible.
+        "size": float,     // Modifier to scale the texture (grid mesh size).
+        "offset": float,   // The Vertical offset of the grid in meters.
+        "falloff": float,  // The size of the plane, the grid is drawn on.
+        "texture": string, // The path to the texture used for the grid ("../share/resources/textures/gridCrossSmall.png", ".../gridCrossSmall.png", ".../gridCentered.png").
+        "alpha": float,    // The transparency of the grid.
+        "color": string    // The color of the grid (as a hex-code string, i.e. #ffffff).
+      },
+      "vignette": {
+        "enabled": bool,                 // Toggle whether the FoV Vignette should be visible.
+        "debug": bool,                   // Toggle whether the Vignette is always drawn with its minimum radius.
+        "innerRadius": float,            // The inner radius of the vignette where the inside is 100% transparent.
+        "outerRadius": float,            // The outer radius of the vignette after which the vignette is 100% opaque.
+        "color": string,                 // The color of the vignette (as a hex-code string).
+        "fadeDuration": float,           // The time it takes for the animation to fade in the vignette in seconds.
+        "fadeDeadzone": float,           // The time of movement above the velocity threshold that is needed before the animation is played in seconds.
+        "lowerVelocityThreshold": float, // The lower threshold below which the vignette will not display (relative to Spice frame from 0..~10). 
+        "upperVelocityThreshold": float, // The upper threshold above which the vignette is always at ist minimum radius (relative to Spice frame from 0..~10).
+        "useDynamicRadius": bool,        // Toggle whether to use the dynamic vignette radius, or the fade animation with fixed radii.
+        "useVerticalOnly": bool          // Toggle whether to only draw the vignette horizontally and keep the sides unobstructed.
+      }
```

Comment:
As it's not enabled ber default, I would suggest to add a copy-paste example configuration to the README with some proper default values.

Reply:
added in commit 4f6d25c